### DISCUSSION TITLE
QST: Read multiple geometry columns from PostGIS

### DISCUSSION THREAD

**Discussion by chriswizz** [geopandas/geopandas#2713] [Q&A]
- [x] I have searched the [geopandas] tag on [StackOverflow](https://stackoverflow.com/questions/tagged/geopandas) and [GIS StackExchange](https://gis.stackexchange.com/questions/tagged/geopandas) for similar questions.

- [ ] I have asked my usage related question on [StackOverflow](https://stackoverflow.com) or [GIS StackExhange](https://gis.stackexchange.com).

---

#### Question about geopandas

I have a PostGIS table that contains multiple geometry columns but I can only read one of them as a geometry column with geopandas. Is there a way to read multiple geometry columns from PostGIS to a GeoDataFrame?

something like a list of geometry columns:

```python
geopandas.GeoDataFrame.from_postgis(sql, con, geom_col=["col_1", "col_2"])

```

Thank you

**Comment by martinfleis** [comment]
Are the other columns ignored or read as WKB?
